src/Core/Services/Implementations/OrganizationService.cs

```diff
@@ -1559,7 +1558,7 @@ public async Task<OrganizationUser> AcceptUserAsync(string orgIdentifier, User u
             organization.MaxAutoscaleSeats.HasValue &&
             organization.MaxAutoscaleSeats.Value < organization.Seats.Value + seatsToAdd)
         {
-            return (false, $"Cannot invite new users. Seat limit has been reached.");
+            return (false, $"Seat limit has been reached.");
```

Comment:
Quick note on this change: no other error message in the autoscaling code refers to the operation the user is trying to undertake (inviting or restoring users), so removing this reference seemed the most pragmatic way to make it reusable between different types of operations.